stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -412,6 +412,14 @@ extension Unsafe${Mutable}RawBufferPointer {
     guard let position = _position else {
       return
     }
+
+    if source.withContiguousStorageIfAvailable({
+      (buffer: UnsafeBufferPointer<C.Element>) in
+      position.copyMemory(from: buffer.baseAddress!, byteCount: source.count)
```

Comment:
What happens if `source` (that is, `buffer`) is empty? Depending on how `wCSIA` is implemented for this particular Collection, `baseAddress` could be `nil`.